projects/npm-tools/packages/npm-scripts/src/typescript/runTSC.js

```diff
@@ -62,4 +70,43 @@ function runTSC() {
 	return true;
 }
 
+async function postProcess({compilerOptions: {outDir: baseDir}}) {
+	const paths = expandGlobs(['**/*.d.ts'], [], {baseDir});
+
+	if (!paths.length) {
+		return;
+	}
+
+	const config = getMergedConfig('prettier');
+
+	for (const filepath of paths) {
+		try {
+			const source = fs.readFileSync(filepath, 'utf8');
+
+			const prettierOptions = {
+				...config,
+				filepath,
+			};
+
+			const formatted = await prettier.format(source, prettierOptions);
+
+			// Fix missing trailing space after license header.
+
+			const fixed = formatted.replace(
+				/^\/\*\*\n(?: \*[^\n]*\n)* \*\/\n[^\n]/,
```

Comment:
Sorry about the readability here, but I don't think we want to go as far as doing an AST-based transform for this little detail.

Here's a translation to make it more obvious what's going on here:

<img width="961" alt="Screenshot 2021-04-07 -182032-X4lSuSLZ@2x" src="https://user-images.githubusercontent.com/7074/113900301-f76ae600-97cd-11eb-8ae0-7d83c5273db4.png">

Reply:
You should've started with this, now it makes total sense